go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/xmidt-org/wrp-go/v4
+module github.com/xmidt-org/wrp-go/v5
 
 go 1.21
```

Comment:
I ran `go mod tidy` locally and it removed `github.com/k0kubun/pp` since it's not being used, I pushed it up.

Just wanted to drop a note.

fyi, @schmidtw